docs/introduction/scope.md

```diff
@@ -1,30 +1,29 @@
 # Scope
 
-Gekko is a free and open source tool that is designed as a starterskit for automated trading on cryptocurrency markets. Gekko aims to have a low barrier entry to writing your own strategies (however a basic scripting knowledge in javascript is required for users who create their own strategies). This page will discuss the scope of some components of Gekko, explaining how things are done and why they are done a certain way.
+Gekko is a free and open source starter kit for automated trading on cryptocurrency markets. Gekko has a low barrier to entry as only minimal knowledge of javascript is required to write trading strategies.
 
 ## Market data
 
-Gekko aggregates all market data into minutely candles (OHLC, VWP and amount of trades). This means Gekko only has to store candles on disk, which take up a predictable amount of space on your harddrive. The tradebot will use some additional market data (the orderbook) to make efficient execute orders efficiently, but this data is not visibile anywhere else.
+Gekko aggregates market data into [OHLC candles](https://en.wikipedia.org/wiki/Open-high-low-close_chart). Gekko only stores candle data on disk taking up a predictable amount of hard drive space. The tradebot will use some additional market data (Such as the orderbook) in order to efficiently execute orders, but this data cached in memory and not accessible.
```

Comment:
Besides OHLC Gekko also exposes VWP and amount of trades metrics, can we keep that in this document?

Reply:
Absolutely.  I tried to google VWP and did not find any documents describing what that is, so I thought it might be confusing for others as well, which is why I took it out.  Ideally there would be a link to an API document that shows how to go about utilizing the VWP and trade volume metrics.  That would also make it clear what they are since the description would be detailed in the API JS Doc / TypeDoc or perhaps a something like [Compodoc](https://medium.com/vincent-ogloblinsky/compodoc-documentation-tool-for-angular-2-applications-44ec650e01a8) would work.